### DISCUSSION TITLE
v4 - important selector

### DISCUSSION THREAD

**Discussion by afzalsayed96** [tailwindlabs/tailwindcss#13779] [General]
Trying to figure out a way to use the `important` selector from the tailwind config in the css and the following seems to work on play with v4.

```css
@import "tailwindcss/preflight" layer(base);

.custom-selector {
  @import "tailwindcss";
  
}
```

Wanted to confirm if this is going to be the correct and official way to achieve the desired behaviour.

**Comment by adamwathan** [comment]
Hey! Not sure how we're going to handle this yet, if that works it's mostly by coincidence at the moment.

**Reply by oschebesta** [reply]
Hi, is it planned to implement this feature into the new version? What would be the v4 way to achieve the same behaviour?

**Reply by wongjn** [reply]
The style of the code in the OP works in v4. Though more concretely, you'd want something more like:

```css
@layer theme, base, components, utilities;

@import "tailwindcss/theme.css" layer(theme);
@import "tailwindcss/preflight.css" layer(base);

.custom-selector {
  @import "tailwindcss/utilities.css" layer(utilities);
}
```

**Reply by oschebesta** [reply]
thanks, that is awesome. 🙌

**Reply by dedalik** [reply]
Not working in tailwind 4

**Reply by wongjn** [reply]
Consider providing a [Tailwind Play](https://play.tailwindcss.com) or a git repo that reproduces the issue and we can take a look for you.

**Reply by dedalik** [reply]
I just copy your code

https://play.tailwindcss.com/s5h8i28vT4?file=css

**Reply by rozsazoltan** [reply]
> Unknown stylesheet: tailwindcss/utiltiies

@dedalik there was typo in the code: `utiltiies` >> `utilities`

https://play.tailwindcss.com/S4sw1ssMbx?file=css

**Reply by dedalik** [reply]
But how it's work, I didn't see it

https://play.tailwindcss.com/89EbG64KuN

Can you create a working example?

**Reply by wongjn** [reply]
https://play.tailwindcss.com/m6OX5Jj6zR

**Reply by dedalik** [reply]
Really works, thank you!
